app/views/shared/_home_navbar.html.erb

```diff
@@ -12,9 +12,9 @@
     <%= link_to url_for( :locale => 'en') do %>
       <%= image_tag "flags/UK_flag.png", class: "flags" %>
     <% end %>
-    
     <%= link_to t(".sign_in", default: "Login"), new_user_session_path, class: "navbar-wagon-item navbar-wagon-link" %>
     <%= link_to t(".sign_up", default: "Sign up"), new_user_registration_path, class: "navbar-wagon-item navbar-wagon-link" %>
+    <%= link_to t(".blog", default: "BLOG"), articles_path, class: "navbar-wagon-item navbar-wagon-link" %>
```

Comment:
i18n defaults are bad, I recommend removing all. in exchange you can config i18n to raise exceptions on not found (see kuina development.rb I think)